src/dataset/csv_dataset.py

```diff
@@ -18,46 +13,34 @@ class CsvType(Enum):
 
 
 class CsvDataset(Dataset):
-    def __init__(
-        self,
-        type: CsvType,
-        csv1: str,
-        csv2: str,
-        links_csv: str,
-        model: BasicModel,
-        rnd: random.Random,
-    ):
-        ids = {}
-        if type == CsvType.products:
-            kg1 = convert_rdf_to_openea(from_csv.convert_products_to_rdf(csv1), ids)
-            kg2 = convert_rdf_to_openea(from_csv.convert_products_to_rdf(csv2), ids)
-        else:
-            kg1 = convert_rdf_to_openea(from_csv.convert_articles_to_rdf(csv1), ids)
-            kg2 = convert_rdf_to_openea(from_csv.convert_articles_to_rdf(csv2), ids)
-        links = pd.read_csv(links_csv)
-        links = [
-            # (ids[parse.quote(str(x[0]))], ids[parse.quote(str(x[1]))], 1)
-            (parse.quote(str(x[0])), parse.quote(str(x[1])))
-            for x in links.to_numpy()
-        ]
-        super().__init__(kg1, kg2, rnd, links)
-        self._kgs = KGs(
-            kg1,
-            kg2,
-            self.labelled_train_pairs,
-            self.labelled_test_pairs,
-            self.labelled_val_pairs,
-            mode=model.args.alignment_module,
-            ordered=model.args.ordered,
+    _download_urls = [
+        "https://speicherwolke.uni-leipzig.de/index.php/s/ro25Tb4WdB6Y4iJ/download"
```

Comment:
This was easier  than to transform them and since its only ~20MB it is no problem